README.md

```diff
@@ -213,6 +213,20 @@ resources in the `"included"` member when the resource names are included in the
   render @posts, include: 'author,comments,comments.author'
 ```
 
+In addition, two types of wildcards may be used. `*` includes one level of associations, and `**` includes all recursively.
+
+```ruby
+  render @posts, include: '**' # or '*' for a single layer
```

Comment:
Cool. Could you add an additional example using nested routes, like `author.comments.posts` and `author.comments.*` so that a) people know the wildcards can be used anywhere, and b) you can add a note saying that when you include `author.comments`, it includes `authors` as well (which makes sense, but might not be clear at first glance).